openapi/integrator/token-requestor/bankaxept.yaml

```diff
@@ -63,6 +63,34 @@ paths:
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
 
+  /v1/eligible-banks:
+    get:
+      summary: Eligible bank identifiers
+      description: From the input of comma-separated list of bank identifiers, return the ones that are eligible for enrolment with ePayment.
+      operationId: eligibleBanks
+      parameters:
+        - name: bankIdentifier
+          description: 4-digit bank identifier
+          in: query
+          schema:
+            items:
+              type: string
+              pattern: "[0-9]{4}"
+              example: "9998"
+            type: array
+          required: true
+      responses:
+        '200':
+          description: Request processed
+          content:
+            application/json;charset=UTF-8:
+              schema:
+                type: string
+                description: Comma-separated list of eligible bank identifiers
+                example: ["9998", "0001"]
```

Comment:
As the content-type is application/json, won't this actually end up as a list embedded in a json string, ie. `"[\"9998\", \"0001\"]"`?

Reply:
See request/response below. Response body was created with ObjectMapper.writeValueAsString() on the list and this is the output. 
For other reviewers, there is a bug with returning a list of strings as the response body in the current openapi-generator version, as it returns then a flux of strings (resulting in horror Mono<ResponseEntity<Flux<String>>>>), so we return a string that is internally a list.